### DISCUSSION TITLE
Invite API Performance Issue

### DISCUSSION THREAD

**Discussion by shiva-rakshith** [Sunbird-RC/community#340] [Issues]
Hello,

We are running registry along with elastic search, postgres and keycloak in AWS. Invite API is taking 5-7 seconds to create a record in the registry and return a response. Is this expected or could it be due to a missing configuration?

cc: @maheshkumargangula

**Comment by ghost** [comment]
Hello,

We are also facing the same issue, for us, it is taking 7 seconds to create a record in the registry and return a response

**Comment by tejash-jl** [comment]
@shiva-rakshith @raman-nd 
can you share your `schema` and the `registry version` used and the `env` configured.

**Reply by ghost** [reply]
`schema` - `default schema provide in documentation`
`registry version` - `v0.0.13`
`env ` -  `default configuration provide in documentation`

**Reply by tejash-jl** [reply]
The first API call might take some time to create the DB tables etc, but the latter ones should be in ms.
@raman-nd can you share your instance configuration

**Reply by ghost** [reply]
Thank you for the clarification

**Reply by shiva-rakshith** [reply]
@tejash-jl - Please find the requested details below:
registry - v0.0.7
schema - https://github.com/Swasth-Digital-Health-Foundation/sunbird-rc-core/blob/hcx-enhancements/java/registry/src/main/resources/public/_schemas/Organisation.json
env - https://github.com/Swasth-Digital-Health-Foundation/sunbird-rc-core/blob/hcx-enhancements/java/registry/src/main/resources/public/_schemas/Organisation.json

Registry Instance Configuration:
Limits:                                                                                                                                         
cpu: 500m                                                   
memory: 1Gi                                                                                                                                                               
Requests:
cpu: 50m
memory: 256Mi

Yes, generally first API call will take more time. But all the API calls are taking 5–6 seconds.
